generation/remap.rsp

```diff
@@ -2518,11 +2524,16 @@ __MIDL___MIDL_itf_mfobjects_0000_0008_0003=MFT_REGISTER_TYPE_INFO
 __MIDL___MIDL_itf_mfobjects_0000_0017_0001=MF_FILE_ACCESSMODE
 __MIDL___MIDL_itf_mfobjects_0000_0017_0002=MF_FILE_OPENMODE
 __MIDL___MIDL_itf_mfobjects_0000_0017_0003=MF_FILE_FLAGS
+__MIDL___MIDL_itf_mmdeviceapi_0000_0000_0001=EDataFlow
+__MIDL___MIDL_itf_mmdeviceapi_0000_0000_0002=ERole
+__MIDL___MIDL_itf_mmdeviceapi_0000_0000_0003=EndpointFormFactor
+__MIDL___MIDL_itf_mmdeviceapi_0000_0008_0001=AudioExtensionParams
 __MIDL___MIDL_itf_strmif_0000_0026_0002=Windows
 __MIDL___MIDL_itf_strmif_0000_0132_0001=VALID_UOP_FLAG
 __MIDL___MIDL_itf_strmif_0000_0132_0002=DVD_CMD_FLAGS
 __MIDL___MIDL_itf_strmif_0000_0132_0003=DVD_OPTION_FLAG
 __MIDL___MIDL_itf_strmif_0000_0132_0004=DVD_RELATIVE_BUTTON
 __MIDL___MIDL_itf_strmif_0000_0138_0001=AM_DVD_RENDERSTATUS
+__MIDL___MIDL_itf_wtypes_0000_0001_0001::__MIDL___MIDL_itf_wtypes_0000_0001_0005=_tagged_union_e__Struct
```

Comment:
This should already be handled by https://github.com/terrafx/terrafx.interop.windows/pull/108/files#diff-8545e26d2a6a186caa7642414b034394R2539
```suggestion
```